### PR TITLE
Spt 482:Update vocab schema to add extra validation

### PR DESCRIPTION
**Pull request by chaitanguttarlapalli**
**What:**

Differences have been identified between the SPOT MVP credential schema and the schema published in the [vocab repo]. Update the vocab site, to reflect the additions as detailed in the acceptance criteria.

**Why:**
The schema published in the vocab repo should reflect the SPOT checks, such that any CRI checking their work against schema will not be caught by any additional validation enforced by the SPOT schema.
We are updating the vocab schema so it becomes the source of truth and, ultimately, will be using the vocab schema in SPOT instead of the current MVP schema.

**AcceptanceCriteria:**
The following changes to the schema have been made:
$ - Should have the following required claims sub, iss, nbf, and vc
$.jti - Should be a non-empty string uri
$.vtm - Should be a non-empty string uri
$.vc - Should have evidence and type required
$.vc["@context"] -Should be a non-empty string
$.vc.evidence[] - Should require a type and txn 
$.vc.evidence[].activityHistoryScore - Should have a max 4
$.vc.evidence[].ci[] - Should support either null or string
$.vc.evidence[].identityFraudScore -Should have a max max 3
$.vc.evidence[].strengthScore -Should have a max 4
$.vc.evidence[].txn - Should support string and string[]
$.vc.evidence[].validityScore -Should have a max 4
$.vc.evidence[].verificationScore - Should have a max 4
$.vc.credentialSubject[].birthDate.value - Should be required
$.vc.credentialSubject[].name[].nameParts - Should be required

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -21,26 +21,29 @@
       }
     },
     "mkdocs-govuk": {
-      "version": "0.0.1",
+      "license": "MIT",
```
Comment (chris-dco) [inline package-lock.json:24]: Can you remove the package-lock.json changes as they're not relevant to this PR?

File: v1/examples/life-events/death-registered.json [hunk 2]
```diff
@@ -2,6 +2,8 @@
   "iss": "https://life-events-platform.gov.uk.example",
   "iat": 1691637001,
   "jti": "4d3559ec67504aaba65d40b0363faad8",
+  "sub": "urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6",
+  "nbf": 1541493724,
```
Comment (chris-dco) [inline v1/examples/life-events/death-registered.json]: May be worth making this 1691637001. As the issued at time (IAT) is `Thu Aug 10 2023 04:10:01 GMT+0100` and not before (NBF) is `Tue Nov 06 2018 08:42:04 GMT+0000` and to me it seems a little odd having the NBF so far behind the IAT and I am wondering how realistic this is.

File: v1/examples/life-events/death-registration-updated.json [hunk 3]
```diff
@@ -2,6 +2,8 @@
   "iss": "https://life-events-platform.gov.uk.example",
   "iat": 1691637001,
   "jti": "4d3559ec67504aaba65d40b0363faad8",
+  "sub": "urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6",
+  "nbf": 1541493724,
```
Comment (chris-dco) [inline v1/examples/life-events/death-registration-updated.json]: May be worth making this 1691637001. As the issued at time (IAT) is `Thu Aug 10 2023 04:10:01 GMT+0100` and not before (NBF) is `Tue Nov 06 2018 08:42:04 GMT+0000` and to me it seems a little odd having the NBF so far behind the IAT and I am wondering how realistic this is.

File: v1/examples/life-events/death-registered.json (hunk 2 above)
Comment (chaitanguttarlapalli) [reply v1/examples/life-events/death-registered.json]: agree but sadly it only validates format here , I ll get them amended though

File: package-lock.json (hunk 1 above)
Comment (chaitanguttarlapalli) [reply package-lock.json:24]: yes still in draft will do when I get the actual PR out

**Pull request closed without merging**